api-reference/openapi.yaml

```diff
@@ -2735,3 +2735,159 @@ components:
             self:
               type: string
               example: https://api.ctrl-hub.com/v3/governance/schemes/5852cf16-8ec8-4aba-809e-f8d828772c4e/work-orders/5852cf16-8ec8-4aba-809e-f8d828772c4e/permits/5852cf16-8ec8-4aba-809e-f8d828772c4e
+    dataCaptureFormSubmissionVersionModel:
+      type: object
+      properties:
+        type:
+          type: string
+          example: versions
+          enum:
+            - versions
+          description: The resource type
+        id:
+          type: string
+          format: uuid
+          example: 7813209e-db7b-4d3d-82a9-73c3c1b10090
+          description: The ID of the form submission version
+        attributes:
+          type: object
+          properties:
+            status:
+              type: string
+              description: The status of the form submission
+              example: complete
+            content: 
+              type: object
+              description: The content of the submission
+        meta:
+          type: object
+          properties:
+            created_at:
+              type: string
+              example: 2024-06-13T07:15:23Z
+              description: "**Sortable**. The time the operation was created ([RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339))"
+        relationships:
+          type: object
+          properties:
+            form_version:
```

Comment:
we're missing the `form` relationship here

Reply:
we only store the form version against a submission version, so we'd have to reach out to mongo to get the form via the version id - do we want to do this? especially on the GetMany pages?

Reply:
we should already have the submission document in memory - it's how we get the form version and the content. We should get this for free... unless you're using $project aggregations

Reply:
in fact, we should also include a relationship to the submission on the version